### PR TITLE
enh(UI): Refactor some functions into Cypress Custom Commands

### PR DESCRIPTION
**Pull request by SoldierBoy31**
## Description

- Refactor some functions into Cypress Custom Commands
- Fix timeout issue on checkThatConfigurationIsExported function, increase timeout for checkThatFixtureServicesExistInDatabase function

## Type of change

- [X] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [X] 22.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [X] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by bdauria — DISMISSED** [review]

File: tests/e2e/cypress/integration/Resources-status/02-actions/index.ts [hunk 1]
```diff
@@ -56,15 +56,16 @@ When('I select the downtime action on a problematic Resource', () => {
     .find('input[type="checkbox"]:first')
     .click();
 
-  cy.get(`[aria-label="${actions.setDowntime}"]`).last().click();
+  cy.getByLabel({ label: actions.setDowntime, tag: '' }).last().click();
```
Comment (Thebarda) [inline tests/e2e/cypress/integration/Resources-status/02-actions/index.ts]:
```suggestion
  cy.getByLabel({ label: actions.setDowntime }).last().click();
```

File: tests/e2e/cypress/support/index.ts [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import 'cypress-wait-until';
-
-import { setUserTokenApiV1 } from './centreonData';
+import './Commands';
 
 before(() => {
   return cy
     .exec(`npx wait-on ${Cypress.config().baseUrl}`)
-    .then(setUserTokenApiV1);
+    .then(() => cy.setUserTokenApiV1());
```
Comment (Thebarda) [inline tests/e2e/cypress/support/index.ts]:
```suggestion
    .then(cy.setUserTokenApiV1);
```

File: tests/e2e/cypress/integration/Resources-status/02-actions/index.ts (hunk 1 above)
Comment (SoldierBoy31) [reply tests/e2e/cypress/integration/Resources-status/02-actions/index.ts]: Update on commit 9e5c024

File: tests/e2e/cypress/support/index.ts (hunk 2 above)
Comment (SoldierBoy31) [reply tests/e2e/cypress/support/index.ts]: Update on commit 9e5c024

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**